File: pgm/ABC_C#/ABC_158_B.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        long N = long.Parse(arr[0]);
        long A = long.Parse(arr[1]);
        long B = long.Parse(arr[2]);
        long dv, rm;
        dv = N / (A + B);
        rm = N % (A + B);
        Console.WriteLine(A * dv + Math.Min(A, rm));
    }
}

File: pgm/ABC_Visual_Basic/ABC_158_B.vb
Class Main
    Public Shared Sub Main()
        Dim N, A, B As Long
        Dim arr() As String
        arr = Split(Console.ReadLine())
        N = arr(0) : A = arr(1) : B = arr(2)
        Dim dv, rm As Long
        dv = Math.Floor(N / (A + B))
        rm = N Mod (A + B)
        Console.WriteLine(A * dv + Math.Min(A, rm))
    End Sub
End Class